DexcomCGMKit/DexcomCGMManagerState.swift

```
@@ -86,6 +94,10 @@ public struct DexcomCGMManagerState: RawRepresentable, Equatable {
             raw[DexcomCGMManagerStateKey.authenticationKeyServiceIdentifier] = authenticationKeyServiceIdentifier
         }
         
+        if let rawAlertConfigurations = try? PropertyListEncoder().encode(alertConfigurations) {
```

[nit] Consider making this:
```
raw[DexcomCGMManagerStateKey.alertConfigurations] = try! PropertyListEncoder().encode(alertConfigurations)
```
so that we will immediately know while developing/testing that something is not `Codable` (as opposed to having it silently fail and having to figure it out the hard way).

---

or call `fatalError()` with a message pointing the dev in the right direction.